spec/_gitignore.txt

```diff
@@ -0,0 +1,33 @@
+#/
```

Comment:
Why is this _gitignore.txt instead of .gitignore?

Reply:
It's to be copied in the `gh-pages` branch. As you suggested in https://github.com/data-apis/array-api/pull/505#issuecomment-1314506373, something to add to a `spec/README.md`, or just document in `_gitignore.txt`.